bundle/edu.gemini.horizons.api/src/main/java/edu/gemini/horizons/api/HorizonsQuery.java

```diff
@@ -90,11 +69,6 @@ private StepUnits(int minValue, int maxValue, String suffix) {
      */
     private Site _site = Site.GN;
 
-    /**
-     * The object type to look.
-     */
-    private ObjectType _type;
```

Comment:
Yes, `HorizonsQuery` and related classes are only used in the PIT, and the `ObjectType` member here was never used except in the `main` method of this class, so I stripped that all out.

The `ObjectType` is used in the `ReplyType`, though, in the `HorizonsResponse`, so I moved it there.